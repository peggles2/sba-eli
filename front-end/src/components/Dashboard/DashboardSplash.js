import React, { Component } from "react";
import { Grid, Header, Button } from "semantic-ui-react";

export default class DashboardSplash extends Component {
  render() {
    return (
      <Grid centered>
        <Grid.Row>
          <Grid.Column mobile={16} tablet={8} computer={8} textAlign="center">
            <Header as="h1">Awesome Hero Caption</Header>
          </Grid.Column>
        </Grid.Row>
        <Grid.Row style={{ fontWeight: "bold" }}>
          <Grid.Column mobile={16} tablet={8} computer={8} textAlign="center">
            <div>
              Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aliquam
              commodo vel est ac tempor. Proin ultricies dictum risus, et tempus
              felis blandit eget. Curabitur ac neque porttitor urna aliquet
              vestibulum nec sit amet enim. Curabitur augue enim, vulputate
              faucibus semper sed, elementum eu arcu. Maecenas id diam sem.
              Nulla dictum eros vitae porta lobortis. Nunc iaculis ullamcorper
              odio. Quisque sed rutrum diam, eget maximus nulla. Nullam sed
              dolor quam. Fusce tempus quis ligula at pharetra. Nam eget erat et
              justo tincidunt laoreet. In finibus massa magna, at pharetra nibh
              aliquam id. Duis aliquet ex sit amet nulla mattis, consequat
              mattis augue porta.
            </div>
          </Grid.Column>
        </Grid.Row>
        <Grid.Row>
          <Button size="huge">Begin Your Journey</Button>
        </Grid.Row>
      </Grid>
    );
  }
}
