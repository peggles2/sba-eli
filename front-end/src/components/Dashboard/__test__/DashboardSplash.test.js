import React from "react";
import DashboardSplash from "../DashboardSplash";
import { Grid } from "semantic-ui-react";
import { shallow } from "enzyme";

describe("DashboardSplash", () => {
  it("should render correctly", () => {
    const wrapper = shallow(<DashboardSplash.WrappedComponent />);

    expect(wrapper).toMatchSnapshot();
  });

  it("should render a <Grid>", () => {
    const wrapper = shallow(<DashboardSplash.WrappedComponent />);

    expect(wrapper.find(Grid).length).toEqual(1);
  });
});
