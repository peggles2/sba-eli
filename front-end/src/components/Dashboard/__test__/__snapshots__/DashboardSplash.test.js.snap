// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DashboardSplash should render correctly 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <DashboardSplash />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {
      "centered": true,
      "children": Array [
        <GridRow>
          <GridColumn
            computer={8}
            mobile={16}
            tablet={8}
            textAlign="center"
          >
            <Header
              as="h1"
              style={
                Object {
                  "color": "#ffffff",
                }
              }
            >
              Awesome Hero Caption
            </Header>
          </GridColumn>
        </GridRow>,
        <GridRow
          style={
            Object {
              "fontWeight": "bold",
            }
          }
        >
          <GridColumn
            computer={8}
            mobile={16}
            tablet={8}
            textAlign="center"
          >
            <div>
              Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aliquam commodo vel est ac tempor. Proin ultricies dictum risus, et tempus felis blandit eget. Curabitur ac neque porttitor urna aliquet vestibulum nec sit amet enim. Curabitur augue enim, vulputate faucibus semper sed, elementum eu arcu. Maecenas id diam sem. Nulla dictum eros vitae porta lobortis. Nunc iaculis ullamcorper odio. Quisque sed rutrum diam, eget maximus nulla. Nullam sed dolor quam. Fusce tempus quis ligula at pharetra. Nam eget erat et justo tincidunt laoreet. In finibus massa magna, at pharetra nibh aliquam id. Duis aliquet ex sit amet nulla mattis, consequat mattis augue porta.
            </div>
          </GridColumn>
        </GridRow>,
        <GridRow>
          <Button
            as="button"
            role="button"
            size="huge"
          >
            Begin Your Journey
          </Button>
        </GridRow>,
      ],
      "style": Object {
        "backgroundColor": "#2d78c9",
      },
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "children": <GridColumn
            computer={8}
            mobile={16}
            tablet={8}
            textAlign="center"
          >
            <Header
              as="h1"
              style={
                Object {
                  "color": "#ffffff",
                }
              }
            >
              Awesome Hero Caption
            </Header>
          </GridColumn>,
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "children": <Header
              as="h1"
              style={
                Object {
                  "color": "#ffffff",
                }
              }
            >
              Awesome Hero Caption
            </Header>,
            "computer": 8,
            "mobile": 16,
            "tablet": 8,
            "textAlign": "center",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "as": "h1",
              "children": "Awesome Hero Caption",
              "style": Object {
                "color": "#ffffff",
              },
            },
            "ref": null,
            "rendered": "Awesome Hero Caption",
            "type": [Function],
          },
          "type": [Function],
        },
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "children": <GridColumn
            computer={8}
            mobile={16}
            tablet={8}
            textAlign="center"
          >
            <div>
              Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aliquam commodo vel est ac tempor. Proin ultricies dictum risus, et tempus felis blandit eget. Curabitur ac neque porttitor urna aliquet vestibulum nec sit amet enim. Curabitur augue enim, vulputate faucibus semper sed, elementum eu arcu. Maecenas id diam sem. Nulla dictum eros vitae porta lobortis. Nunc iaculis ullamcorper odio. Quisque sed rutrum diam, eget maximus nulla. Nullam sed dolor quam. Fusce tempus quis ligula at pharetra. Nam eget erat et justo tincidunt laoreet. In finibus massa magna, at pharetra nibh aliquam id. Duis aliquet ex sit amet nulla mattis, consequat mattis augue porta.
            </div>
          </GridColumn>,
          "style": Object {
            "fontWeight": "bold",
          },
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "children": <div>
              Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aliquam commodo vel est ac tempor. Proin ultricies dictum risus, et tempus felis blandit eget. Curabitur ac neque porttitor urna aliquet vestibulum nec sit amet enim. Curabitur augue enim, vulputate faucibus semper sed, elementum eu arcu. Maecenas id diam sem. Nulla dictum eros vitae porta lobortis. Nunc iaculis ullamcorper odio. Quisque sed rutrum diam, eget maximus nulla. Nullam sed dolor quam. Fusce tempus quis ligula at pharetra. Nam eget erat et justo tincidunt laoreet. In finibus massa magna, at pharetra nibh aliquam id. Duis aliquet ex sit amet nulla mattis, consequat mattis augue porta.
            </div>,
            "computer": 8,
            "mobile": 16,
            "tablet": 8,
            "textAlign": "center",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aliquam commodo vel est ac tempor. Proin ultricies dictum risus, et tempus felis blandit eget. Curabitur ac neque porttitor urna aliquet vestibulum nec sit amet enim. Curabitur augue enim, vulputate faucibus semper sed, elementum eu arcu. Maecenas id diam sem. Nulla dictum eros vitae porta lobortis. Nunc iaculis ullamcorper odio. Quisque sed rutrum diam, eget maximus nulla. Nullam sed dolor quam. Fusce tempus quis ligula at pharetra. Nam eget erat et justo tincidunt laoreet. In finibus massa magna, at pharetra nibh aliquam id. Duis aliquet ex sit amet nulla mattis, consequat mattis augue porta.",
            },
            "ref": null,
            "rendered": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aliquam commodo vel est ac tempor. Proin ultricies dictum risus, et tempus felis blandit eget. Curabitur ac neque porttitor urna aliquet vestibulum nec sit amet enim. Curabitur augue enim, vulputate faucibus semper sed, elementum eu arcu. Maecenas id diam sem. Nulla dictum eros vitae porta lobortis. Nunc iaculis ullamcorper odio. Quisque sed rutrum diam, eget maximus nulla. Nullam sed dolor quam. Fusce tempus quis ligula at pharetra. Nam eget erat et justo tincidunt laoreet. In finibus massa magna, at pharetra nibh aliquam id. Duis aliquet ex sit amet nulla mattis, consequat mattis augue porta.",
            "type": "div",
          },
          "type": [Function],
        },
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "children": <Button
            as="button"
            role="button"
            size="huge"
          >
            Begin Your Journey
          </Button>,
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "as": "button",
            "children": "Begin Your Journey",
            "role": "button",
            "size": "huge",
          },
          "ref": null,
          "rendered": "Begin Your Journey",
          "type": [Function],
        },
        "type": [Function],
      },
    ],
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "centered": true,
        "children": Array [
          <GridRow>
            <GridColumn
              computer={8}
              mobile={16}
              tablet={8}
              textAlign="center"
            >
              <Header
                as="h1"
                style={
                  Object {
                    "color": "#ffffff",
                  }
                }
              >
                Awesome Hero Caption
              </Header>
            </GridColumn>
          </GridRow>,
          <GridRow
            style={
              Object {
                "fontWeight": "bold",
              }
            }
          >
            <GridColumn
              computer={8}
              mobile={16}
              tablet={8}
              textAlign="center"
            >
              <div>
                Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aliquam commodo vel est ac tempor. Proin ultricies dictum risus, et tempus felis blandit eget. Curabitur ac neque porttitor urna aliquet vestibulum nec sit amet enim. Curabitur augue enim, vulputate faucibus semper sed, elementum eu arcu. Maecenas id diam sem. Nulla dictum eros vitae porta lobortis. Nunc iaculis ullamcorper odio. Quisque sed rutrum diam, eget maximus nulla. Nullam sed dolor quam. Fusce tempus quis ligula at pharetra. Nam eget erat et justo tincidunt laoreet. In finibus massa magna, at pharetra nibh aliquam id. Duis aliquet ex sit amet nulla mattis, consequat mattis augue porta.
              </div>
            </GridColumn>
          </GridRow>,
          <GridRow>
            <Button
              as="button"
              role="button"
              size="huge"
            >
              Begin Your Journey
            </Button>
          </GridRow>,
        ],
        "style": Object {
          "backgroundColor": "#2d78c9",
        },
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "children": <GridColumn
              computer={8}
              mobile={16}
              tablet={8}
              textAlign="center"
            >
              <Header
                as="h1"
                style={
                  Object {
                    "color": "#ffffff",
                  }
                }
              >
                Awesome Hero Caption
              </Header>
            </GridColumn>,
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "children": <Header
                as="h1"
                style={
                  Object {
                    "color": "#ffffff",
                  }
                }
              >
                Awesome Hero Caption
              </Header>,
              "computer": 8,
              "mobile": 16,
              "tablet": 8,
              "textAlign": "center",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "as": "h1",
                "children": "Awesome Hero Caption",
                "style": Object {
                  "color": "#ffffff",
                },
              },
              "ref": null,
              "rendered": "Awesome Hero Caption",
              "type": [Function],
            },
            "type": [Function],
          },
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "children": <GridColumn
              computer={8}
              mobile={16}
              tablet={8}
              textAlign="center"
            >
              <div>
                Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aliquam commodo vel est ac tempor. Proin ultricies dictum risus, et tempus felis blandit eget. Curabitur ac neque porttitor urna aliquet vestibulum nec sit amet enim. Curabitur augue enim, vulputate faucibus semper sed, elementum eu arcu. Maecenas id diam sem. Nulla dictum eros vitae porta lobortis. Nunc iaculis ullamcorper odio. Quisque sed rutrum diam, eget maximus nulla. Nullam sed dolor quam. Fusce tempus quis ligula at pharetra. Nam eget erat et justo tincidunt laoreet. In finibus massa magna, at pharetra nibh aliquam id. Duis aliquet ex sit amet nulla mattis, consequat mattis augue porta.
              </div>
            </GridColumn>,
            "style": Object {
              "fontWeight": "bold",
            },
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "children": <div>
                Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aliquam commodo vel est ac tempor. Proin ultricies dictum risus, et tempus felis blandit eget. Curabitur ac neque porttitor urna aliquet vestibulum nec sit amet enim. Curabitur augue enim, vulputate faucibus semper sed, elementum eu arcu. Maecenas id diam sem. Nulla dictum eros vitae porta lobortis. Nunc iaculis ullamcorper odio. Quisque sed rutrum diam, eget maximus nulla. Nullam sed dolor quam. Fusce tempus quis ligula at pharetra. Nam eget erat et justo tincidunt laoreet. In finibus massa magna, at pharetra nibh aliquam id. Duis aliquet ex sit amet nulla mattis, consequat mattis augue porta.
              </div>,
              "computer": 8,
              "mobile": 16,
              "tablet": 8,
              "textAlign": "center",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aliquam commodo vel est ac tempor. Proin ultricies dictum risus, et tempus felis blandit eget. Curabitur ac neque porttitor urna aliquet vestibulum nec sit amet enim. Curabitur augue enim, vulputate faucibus semper sed, elementum eu arcu. Maecenas id diam sem. Nulla dictum eros vitae porta lobortis. Nunc iaculis ullamcorper odio. Quisque sed rutrum diam, eget maximus nulla. Nullam sed dolor quam. Fusce tempus quis ligula at pharetra. Nam eget erat et justo tincidunt laoreet. In finibus massa magna, at pharetra nibh aliquam id. Duis aliquet ex sit amet nulla mattis, consequat mattis augue porta.",
              },
              "ref": null,
              "rendered": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aliquam commodo vel est ac tempor. Proin ultricies dictum risus, et tempus felis blandit eget. Curabitur ac neque porttitor urna aliquet vestibulum nec sit amet enim. Curabitur augue enim, vulputate faucibus semper sed, elementum eu arcu. Maecenas id diam sem. Nulla dictum eros vitae porta lobortis. Nunc iaculis ullamcorper odio. Quisque sed rutrum diam, eget maximus nulla. Nullam sed dolor quam. Fusce tempus quis ligula at pharetra. Nam eget erat et justo tincidunt laoreet. In finibus massa magna, at pharetra nibh aliquam id. Duis aliquet ex sit amet nulla mattis, consequat mattis augue porta.",
              "type": "div",
            },
            "type": [Function],
          },
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "children": <Button
              as="button"
              role="button"
              size="huge"
            >
              Begin Your Journey
            </Button>,
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "as": "button",
              "children": "Begin Your Journey",
              "role": "button",
              "size": "huge",
            },
            "ref": null,
            "rendered": "Begin Your Journey",
            "type": [Function],
          },
          "type": [Function],
        },
      ],
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
